features/custom_product.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Custom product sale
+
+  Background:
+
+         When I'm logged in with 1234     
+
+  Scenario: As a valid user I use the cash payment option        
+
+         Then I press "CartAddNewItem"
+         Then I press 2000
+         Then I press "dialogBaseSaveButton"
+         Then I press "CartGoToPayment"
+         Then I press "$20.00"
+         Then I press "checkoutCloseButton"
+         Then I wait upto 40 seconds for the "ActivityConfirmation" screen to appear
```

Comment:
the wait for the screen to appear seems to be consistent, perhaps eventually we could use a variable to define this